src/FieldTrait.php

```diff
@@ -22,14 +22,35 @@ trait FieldTrait
     public function setSelectionSet(array $selectionSet)
```

Comment:
@actionm I kind of don't understand why this change is necessary? :thinking: 
Is it because you want to add the feature of selecting fields with aliases and arguments through the `Query` class? It's not required for selecting one field through the `AbstractQueryBuilder` as it converts all selected fields into string before the selection set is set in the query class. Or maybe I'm missing something here.